### PR TITLE
fix(issuer): fix styles for whitelist modal

### PR DESCRIPTION
**Pull request by sajclarke**
**Please make sure the following boxes are checked before submitting your Pull Request:**

- [ ] I linked the issues related to this PR in its description (if any).
- [ ] If this PR adds new code that is not going to change in the near future, it includes unit tests to cover it.

### CODE REVIEW

File: packages/polymath-issuer/src/pages/compliance/components/ImportWhitelistModal.js [hunk 1]
```diff
@@ -3,13 +3,13 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import {
-  Modal,
+  // Modal,
```
Comment (RafaelVidaurre) [inline packages/polymath-issuer/src/pages/compliance/components/ImportWhitelistModal.js]: 👁

Comment (sajclarke) [reply packages/polymath-issuer/src/pages/compliance/components/ImportWhitelistModal.js]: Resolved

Comment (Grsmto) [reply packages/polymath-issuer/src/pages/compliance/components/ImportWhitelistModal.js]: Why leaving it as commented?

Comment (sajclarke) [reply packages/polymath-issuer/src/pages/compliance/components/ImportWhitelistModal.js]: I already fixed this in the latest commit

**Comment by RafaelVidaurre** [comment]
@sajclarke all good, please solve conflicts and ping me

**Pull request merged**